src/server/plugins/engine/components/UkAddressField.ts

```diff
@@ -159,7 +170,8 @@ export class UkAddressField extends FormComponent {
       isFormState(value) &&
       TextField.isText(value.addressLine1) &&
       TextField.isText(value.town) &&
-      TextField.isText(value.postcode)
+      TextField.isText(value.postcode) &&
+      (value.county === undefined || TextField.isText(value.county))
```

Comment:
I'm unsure why this line is different to the others for `addressLine1`, `town` and `postcode`?
Also, could the line go above postcode for consistency?